src/aliceVision/image/io.hpp

```diff
@@ -42,6 +42,20 @@ enum class EImageFileType
   EXR
 };
 
+/**
+ * @brief aggregate for multiple image reading options
+ */
+struct SImageReadOptions
```

Comment:
I would remove the S prefix as we are not using it for any other struct.
```suggestion
struct ImageReadOptions
```